site/sfguides/src/data_mapping_in_native_apps/data_mapping_in_native_apps.md

```diff
@@ -221,11 +312,12 @@ CREATE OR ALTER VERSIONED SCHEMA ENRICHIP;
 GRANT USAGE ON SCHEMA ENRICHIP TO APPLICATION ROLE APP_PUBLIC;
 
 --this is an application version of the object shared with the application package
-CREATE VIEW ENRICHIP.LITEDB11 AS SELECT * FROM IP2LOCATION.litedb11;
+CREATE VIEW IF NOT EXISTS ENRICHIP.LITEDB11 AS SELECT * FROM IP2LOCATION.litedb11;
+-- If the user prefers, access can be granted directly to the IPLOCATION.litebd11 view, created in the setup_package_script file.
```

Comment:
Because this is going in a versioned schema I suggest `CREATE OR REPLACE` as a best practice. The only reason it might be necessary is when an ALTER APPLICATION UPGRADE or CREATE APPLICATION fails, in which case the next attempt at installation will use the same versioned schema "containers". We likely want the versions from the more-recent attempt to overwrite any objects created from earlier instances; correctness here is more important than making sure the subsequent setup is as fast as possible.